ui/redux/selectors.js

```diff
@@ -428,9 +433,22 @@ export const getRnaSeqSignificantJunctionData = createSelector(
           geneSymbol: (genesById[geneId] || {}).geneSymbol || geneId,
           idField: `${geneId}-${chrom}-${start}-${end}-${strand}-${type}`,
           familyGuid: individualsByGuid[individualGuid].familyGuid,
+          individualName: individualsByGuid[individualGuid].displayName,
           individualGuid,
           ...{ geneId, chrom, start, end, strand, type, ...cols },
         })),
     } : acc), {},
   ),
 )
+
+export const getSpliceOutliersByChromFamily = createSelector(
+  getRnaSeqSignificantJunctionData,
+  spliceDataByIndiv => Object.values(spliceDataByIndiv).reduce(
+    (acc, spliceData) => {
+      if (!spliceData || spliceData.length < 1) {
```

Comment:
It would be better if `getRnaSeqSignificantJunctionData` had a filter in the logic to prevent empty values in the first place